src/main/java/io/stargate/sgv2/jsonapi/config/DebugConfigAccess.java

```diff
@@ -0,0 +1,25 @@
+package io.stargate.sgv2.jsonapi.config;
+
+import com.google.common.annotations.VisibleForTesting;
+
+/**
+ * Static accessor for the debug mode configuration; initialized on startup, used to work around
+ * access issues with the Quarkus config system (can access without needing to inject the config
+ * interface).
+ */
+public class DebugConfigAccess {
+  private static boolean debugEnabled;
+
+  public static void initialize(DebugModeConfig config) {
```

Comment:
nit: pass in a boolean so it does not have a dependency on the config structure

Reply:
I was thinking about that but was bit worried about method overload, this being more type safe. But I can rename the method I guess, to make it more clear.

Reply:
Changed as suggested: will just use the method `setDebugEnabled` that is used by tests.